dev-infrastructure/modules/aks-cluster-base.bicep

```diff
@@ -253,7 +253,7 @@ resource aksCluster 'Microsoft.ContainerService/managedClusters@2024-01-01' = {
         enabled: true
         config: {
           enableSecretRotation: 'true'
-          rotationPollInterval: '24h'
+          rotationPollInterval: '5m'
```

Comment:
Is this an intentional change? The change makes sense to me, I don't know why it was 24h previously, but just want to make sure

Reply:
we can discuss around the value. i think the default is 2m